rpc/client_server_test.go

```diff
@@ -218,6 +220,7 @@ func TestGetOrdersSuccess(t *testing.T) {
 	assert.Len(t, getOrdersResponse.OrdersInfos, 1)
 
 	assert.Equal(t, returnedSnapshotID, getOrdersResponse.SnapshotID, "SnapshotID did not match")
+	assert.Equal(t, expectedSnapshotTimestamp, getOrdersResponse.SnapshotTimestamp, "SnapshotTimestamp did not match")
```

Comment:
If you get unlucky with the timing of this test, the timestamp in the response can be one second after `expectedSnapshotTimestamp`, which would cause the test to fail. Can we use [`assert.WithinDuration`](https://godoc.org/github.com/stretchr/testify/assert#WithinDuration) to account for this?